tests/test_formatters.py

```diff
@@ -174,6 +174,18 @@ def test_percentage_format(env: LoggingEnvironment, class_: type[BaseJsonFormatt
     return
 
 
+@pytest.mark.parametrize("class_", ALL_FORMATTERS)
+def test_defaults_field(env: LoggingEnvironment, class_: type[BaseJsonFormatter]):
+    env.set_formatter(class_(defaults={"first": 1, "second": 2}))
+
+    env.logger.info("testing defaults field", extra={"first": 1234})
+    log_json = env.load_json()
+
+    assert log_json["first"] == 1234
+    assert log_json["second"] == 2
+    return
+
```

Comment:
We may want to add a test that overrides fields when passing a dict as a message (instead of a string).